webapp/packages/core-administration/src/AdministrationScreen/AdministrationScreenService.ts

```diff
@@ -7,10 +7,10 @@
  */
 import { computed, makeObservable, observable } from 'mobx';
 
-import { EAdminPermission } from '@cloudbeaver/core-authentication';
 import { injectable } from '@cloudbeaver/core-di';
 import { NotificationService } from '@cloudbeaver/core-events';
 import { Executor, IExecutor } from '@cloudbeaver/core-executor';
+import { EAdminPermission } from '@cloudbeaver/core-resource';
```

Comment:
core-resource is not responsible for gql schema related things, it's better to move EAdminPermission to core-sdk / core-root

Reply:
moved to `core-root`